edx_django_utils/monitoring/internal/middleware.py

```diff
@@ -285,6 +286,19 @@ def __call__(self, request):
 
         response = self.get_response(request)
 
+        # .. setting_name: COOKIE_PREFIXES_TO_REMOVE
+        # .. setting_default: []
+        # .. setting_description: A list of cookie prefixes. Any cookie starting with a prefix in this list
+        # will be manually removed from responses (i.e. set to expire in the past)
+        deprecated_cookie_prefixes = getattr(settings, "COOKIE_PREFIXES_TO_REMOVE", [])
+        yesterday = datetime.utcnow() - timedelta(days=1)
+        for cookie_name in request.COOKIES.keys():
+            for deprecated_cookie_prefix in deprecated_cookie_prefixes:
+                if cookie_name.startswith(deprecated_cookie_prefix):
+                    log.info(f"Forcing expiration of {cookie_name}")
+                    response.set_cookie(cookie_name, value='', expires=yesterday,
+                                        domain=settings.BASE_COOKIE_DOMAIN)
```

Comment:
Hah. I did see that but ignored it since I thought it did something different and that we specifically wanted to expire. Will update.

Reply:
Yeah, "delete" is... a little misleading. Cookies are super weird.

Reply:
Hah. I did see that but ignored it since I thought it did something different and that we specifically wanted to expire. Should have looked more carefully. Will update.

Reply:
[fixed]